cmd/finch/nerdctl_remote.go

```diff
@@ -157,25 +170,38 @@ func (nc *nerdctlCommand) run(cmdName string, args []string) error {
 					envs = append(envs, addEnv)
 				}
 			case shortFlagBoolSet.Has(arg) || longFlagBoolSet.Has(arg):
-				// exact match to a short flag: -?
+				// exact match to a short no argument flag: -?
 				// or exact match to: --<long_flag>
 				nerdctlArgs = append(nerdctlArgs, arg)
+			case longFlagBoolSet.Has(strings.Split(arg, "=")[0]):
```

Comment:
we can add this to the or statement to the above case

Reply:
All of the switch case statements were intentionally broken out for explicit commenting for future maintenance.
Nevertheless, based on your feedback and feedback from Justin, I am currently refactoring the "handlers" and "processing loop" to move away from the large quantity of processing case scenarios.